executorCommon/src/main/scala/dx/executor/JobMeta.scala

```diff
@@ -55,6 +56,7 @@ object JobMeta {
   // this file has all the information about the executable
   val ExecutableInfoFile = "dnanexus-executable.json"
   val MaxConcurrentUploads = 8
+  val MaxManifestJsLength = 100_000
```

Comment:
Could you add a comment on what this length is (number of chars)?

Reply:
done